src/main/webapp/WEB-INF/view/subscription.jsp

```diff
@@ -0,0 +1,15 @@
+<%@ page import="java.util.List" %>
+<%@ page import="java.util.ArrayList" %>
+
+<!DOCTYPE html>
```

Comment:
Do we want this page if the user can see their subscriptions on the profile page?

Reply:
I can move it I just wasn't sure where

Reply:
I meant why don't we remove it entirely, since we already show the subscriptions on the profile page.

Reply:
I can do that